.chloggen/38619-add-config.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
```

Comment:
Since it is not operational yet I don't think we need a changelog - I added the skip changelog label so removing it is not going to fail CI

Reply:
done!